beta-src/src/components/ui/WDSaveReadyButtons.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from "react";
+
+import { Stack } from "@mui/material";
+import WDButton from "./WDButton";
+import SaveStatus from "../../types/saveStatus";
+
+interface saveReadyProps {
```

Comment:
interfaces are TitleCase...so SaveReadyProps. i still prefer this component be called MoveControls or something like that because suppose Save and Ready change then the component name won't make sense any more